### PR TITLE
Implementing servicing for CoreCLR runtime

### PR DESCRIPTION
**Pull request by moozzyk**
@victorhurdugaci @muratg @davidfowl

### CODE REVIEW

File: src/dnx/servicing.cpp [hunk 1]
```diff
@@ -31,6 +31,10 @@ namespace
     {
         const std::string runtime_qualifier = std::string{ "dnx|" } + runtime_moniker + "|" + ProductVersionStr + "=";
 
+        trace_writer.write(
+            std::wstring(L"Looking for redirections for runtime ")
+                .append(dnx::utils::to_wstring(runtime_qualifier).substr(0, runtime_qualifier.length() - 1)), true);
+
```
Comment (muratg) [inline src/dnx/servicing.cpp]: :+1:

**Comment by muratg** [comment]
Code looks good. I assume you already tested it. :)

:shipit:

**Comment by davidfowl** [comment]
I think trace is still super hard to read :smile:. The rest of the code looks good tho

**Comment by moozzyk** [comment]
@muratg - Yes, I debugged through different scenarios

@davidfowl - there is no string interpolation in C++ ;) - I do have an idea which I need to try out but it will be probably after beta6...

**Pull request merged**